src/main/java/core/basesyntax/FileReader.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+
+public interface FileReader {
+
+    List<String> readFromFile(String filePath) throws FileNotFoundException;

```

Comment:
```suggestion
    List<String> readFromFile(String filePath);
```
Don’t  throw exception in the method signature